### DISCUSSION TITLE
What's the meanings to the output line prefix characters?

### DISCUSSION THREAD

**Discussion by Thell** [ERGO-Code/HiGHS#1859] [Q&A] [answered]
Is it documented anywhere as to what the different console line prefix characters mean? I've seen quite a few but the only thing I know is when I see the 'L' prefix I have an improved solution. It'd be great to see a list somewhere. 

Just to be clear...

```
        Nodes      |    B&B Tree     |            Objective Bounds              |  Dynamic Constraints |       Work      
     Proc. InQueue |  Leaves   Expl. | BestBound       BestSol              Gap |   Cuts   InLp Confl. | LpIters     Time

 T   10153     834      3572  46.47%   146585110.7583  136171093.0001     7.65%     3723    585   9628     8529k  6635.7s
```

The `T` in that line is what I am calling the prefix character (or message type?). Searching for `\"L\"` in the code base didn't get me to the info.

**Comment by jajhall** [comment, marked as answer]
See #1736
